packages/ui-kit/src/components/rss-feeds.js

```diff
@@ -1,121 +1,86 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import styled from '@emotion/styled';
-import css from '@emotion/css';
 import useSWR from 'swr';
 import Parser from 'rss-parser';
-import moment from 'moment';
+import PropTypes from 'prop-types';
 import LoadingSpinner from '@commercetools-uikit/loading-spinner';
 import ContentNotifications from './content-notifications';
-import { colors, tokens, dimensions } from '../design-system';
-import Link from './link';
-
-const Table = styled.table`
-  width: 100%;
-  border-radius: ${tokens.borderRadiusForRssFeedTable};
-  box-shadow: ${tokens.shadowForRssFeedTable};
-
-  thead {
-    th {
-      background-color: ${colors.light.surfaceSecondary2};
-      border-radius: ${tokens.borderRadiusForRssFeedTable}
-        ${tokens.borderRadiusForRssFeedTable} 0 0;
-      padding: ${dimensions.spacings.s};
-    }
-  }
+import RssFeedTable from './rss-feed-table';
 
-  tbody {
-    tr {
-      td {
-        padding: ${dimensions.spacings.m};
+async function fetcher(...args) {
+  const rssParser = new Parser();
+  const promises = args.map(async (feed) => {
+    const feedData = await rssParser.parseURL(feed.url);
+    const refactoredData = feedData.items.map((item) => {
+      return { ...item, feedName: feed.title };
+    });
+    return refactoredData;
+  });
+  return Promise.all(promises);
+}
 
-        :first-of-type {
-          color: ${colors.light.linkHover};
-        }
+const transformData = (data) => {
```

Comment:
This function has some "complex" logic. It's better to provide some unit tests here, to make sure the logic is solid.

Reply:
Hmm let me know if you would add some other unit tests as well...
4c633a2